config.json

```diff
@@ -1160,6 +1160,14 @@
           "strings"
         ]
       },
+      {
+        "slug": "game-of-life",
+        "name": "Conway's Game of Life",
+        "uuid": "eae82e30-b6c8-4414-8b56-902da5d61f32",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 3
```

Comment:
Can we increase the difficulty to Medium (4..7)

I think of this exercise as being similar in difficulty to minesweeper. The Lua track currently gives minesweeper a difficulty of 7.

Tracks often give both a difficulty around 5,

Reply:
I re-ranked it as a 5. Knapsack is marked as a 6 and Knapsack is a more difficult problem, IMO.